Userland/Applications/Browser/CookieJar.cpp

```diff
@@ -48,9 +49,9 @@ void CookieJar::set_cookie(const URL& url, const Web::Cookie::ParsedCookie& pars
 
 void CookieJar::dump_cookies() const
 {
-    static const char* key_color = "\033[34;1m";
-    static const char* attribute_color = "\033[33m";
-    static const char* no_color = "\033[0m";
+    constexpr StringView key_color = "\033[34;1m";
+    constexpr StringView attribute_color = "\033[33m";
+    constexpr StringView no_color = "\033[0m";
```

Comment:
I think using operator""sv would be helpful for performance (since the strlen call would be eliminated), aka:
```suggestion
    constexpr StringView key_color = "\033[34;1m"sv;
    constexpr StringView attribute_color = "\033[33m"sv;
    constexpr StringView no_color = "\033[0m"sv;
```
(if you decide to go with this, theres a couple other instances in your commit :)

Reply:
@IdanHo Thanks for the suggestion. This would not make a difference though. The `sv` calls the constructor which has to do a strlen (or sizeof) and this is all done at compile-time. There is no run-time effect.